api/tests/domain/demarches_simplifiees_test.py

```diff
@@ -179,137 +179,159 @@ def test_raise_error_in_unknown_state(self):
+    "codeEffectifEntreprise": "12",
+    "dateCreation": "2001-06-27",
+    "formeJuridique": "SAS, " "société " "par " "actions " "simplifiée",
+    "formeJuridiqueCode": "5710",
+    "inlineAdresse": "2 " "RUE " "LAMENNAIS, " "75008 " "PARIS " "8",
+    "nom": None,
+    "nomCommercial": "",
+    "numeroTvaIntracommunautaire": "FR67438391195",
+    "prenom": None,
+    "raisonSociale": "GAUMONT " "ALESIA",
+    "siren": "438391195",
+    "siretSiegeSocial": "43839119500056",
+}
+EXPECTED_RESULT = {
+    "status": "en_construction",
+    "updated_at": "2021-11-12T14:51:42+01:00",
+    "firstname": "John",
+    "lastname": "Doe",
+    "phone_number": "0102030405",
+    "siret": "43839119500056",
+    "siren": "438391195",
+    "iban": "FR7630001007941234567890185",
+    "bic": "QSDFGH8Z",
+    "annotation_id": "InterestingId",
+    "dossier_id": "Q2zzbXAtNzgyODAw",
+}
+DMS_TOKEN_FIELD_RESULT = {"dms_token": "50a7536a21c8"}
+
+
 class ParseRawBicDataTest:
     @pytest.mark.parametrize(
-        "entreprise",
+        "entreprise, identifiant_du_lieu, expected_result",
         [
-            {
-                "capitalSocial": "38130",
-                "codeEffectifEntreprise": "12",
-                "dateCreation": "2001-06-27",
-                "formeJuridique": "SAS, " "société " "par " "actions " "simplifiée",
-                "formeJuridiqueCode": "5710",
-                "inlineAdresse": "2 " "RUE " "LAMENNAIS, " "75008 " "PARIS " "8",
-                "nom": None,
-                "nomCommercial": "",
-                "numeroTvaIntracommunautaire": "FR67438391195",
-                "prenom": None,
-                "raisonSociale": "GAUMONT " "ALESIA",
-                "siren": "438391195",
-                "siretSiegeSocial": "43839119500056",
-            },
-            None,
+            (COMPANY_FIELD, None, EXPECTED_RESULT),
+            (COMPANY_FIELD, VENUE_DMS_TOKEN_FIELD, {**EXPECTED_RESULT, **DMS_TOKEN_FIELD_RESULT}),
+            (None, None, EXPECTED_RESULT),
+            (None, VENUE_DMS_TOKEN_FIELD, {**EXPECTED_RESULT, **DMS_TOKEN_FIELD_RESULT}),
         ],
     )
-    def test_parsing_works(self, entreprise):
-        INPUT_DATA = {
+    def test_parsing_works(self, entreprise, identifiant_du_lieu, expected_result):
+        champs = [
+            {"id": "Q2hhbXAtNDA3ODk1", "label": "Mes informations", "stringValue": "", "value": None},
+            {"id": "Q2hhbXAtNDA3ODg5", "label": "Mon prénom", "stringValue": "John", "value": "John"},
+            {"id": "Q2hhbXAtNDA3ODkw", "label": "Mon nom", "stringValue": "Doe", "value": "Doe"},
+            {
+                "id": "Q2hhbXAtNDA3ODky",
+                "label": "Mon numéro de téléphone",
+                "stringValue": "01 02 03 04 05",
+                "value": "0102030405",
+            },
+            {
+                "id": "Q2hhbXAtODU2NzEz",
+                "label": "Informations relatives au responsable légal " "et à ma délégation de gestion financière",
```

Comment:
(style / micro-détail) Black (je suppose) nous fait des blagues : 

```suggestion
                "label": "Informations relatives au responsable légal et à ma délégation de gestion financière",
```

idem plus bas.